src/bin/solang.rs

```diff
@@ -878,19 +858,45 @@ fn imports_arg(matches: &ArgMatches) -> FileResolver {
         }
     }
 
-    if let Some(maps) = matches.get_many::<String>("IMPORTMAP") {
-        for p in maps {
-            if let Some((map, path)) = p.split_once('=') {
-                if let Err(e) = resolver.add_import_map(OsString::from(map), PathBuf::from(path)) {
-                    eprintln!("error: import path '{}': {}", path, e);
-                    std::process::exit(1);
-                }
-            } else {
-                eprintln!("error: import map '{}': contains no '='", p);
+    if let Some(maps) = matches.get_many::<(String, String)>("IMPORTMAP") {
+        for (map, path) in maps {
+            if let Err(e) = resolver.add_import_map(OsString::from(map), PathBuf::from(path)) {
+                eprintln!("error: import path '{}': {}", path, e);
                 std::process::exit(1);
             }
         }
     }
 
     resolver
 }
+
+fn parse_import_map(map: &str) -> Result<(String, String), String> {
+    if let Some((var, value)) = map.split_once('=') {
+        Ok((var.to_owned(), value.to_owned()))
+    } else {
+        Err("contains no '='".to_owned())
+    }
+}
+
+#[derive(Serialize)]
+pub struct EwasmContract {
```

Comment:
Why did you move these structs down?

Reply:
These structs are the serialization format for `--standard-json`. I don't think they belong at the top of the `main.rs` for the executable. Where do you think they should go?

Reply:
Perhaps they belong in another file.

Reply:
You are right. It was even worse: the structures were defined in different places. I've created `src/standard_json.rs`.